modules/elasticache-redis/remote-state.tf

```diff
@@ -0,0 +1,30 @@
+module "vpc" {
+  source  = "cloudposse/stack-config/yaml//modules/remote-state"
+  version = "0.13.0"
+
+  component               = "vpc"
+  stack_config_local_path = "../../../stacks"
+
+  context = module.this.context
+}
+
+module "eks" {
+  source  = "cloudposse/stack-config/yaml//modules/remote-state"
+  version = "0.13.0"
+
+  stack_config_local_path = "../../../stacks"
+  component               = "eks"
+
+  context = module.this.context
+}
```

Comment:
This is going to bite us... since we also support ECS. Anyways, guess we'll need to generalize this later.